Can't compile (enable Travis CI?)

```
>gradlew.bat
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Unable to start the daemon process.
This problem might be caused by incorrect configuration of the daemon.
For example, an unrecognized jvm option is used.
Please refer to the user guide chapter on the daemon at https://docs.gradle.org/3.3/userguide/gradle_daemon.html
Please read the following process output to find out more:
-----------------------
Error occurred during initialization of VM
Could not reserve enough space for 1572864KB object heap

> Starting Daemon
```

Maybe provide Travis builds? I can send you PR if you enable it at https://travis-ci.org/nitroshare/

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,1 @@
+language: android

